src/python/grpcio/grpc/_adapter/_types.py

```diff
@@ -321,6 +354,28 @@ def create_call(self, completion_queue, method, host, deadline=float('+inf')):
     """
     return None
 
+  @abc.abstractmethod
+  def connectivity_state(self, try_to_connect=False):
```

Comment:
Please avoid optional parameters on internal APIs. It's no trouble to us to always be explicit about passing the parameter all the time.